spec/openapi.yaml

```diff
@@ -9489,6 +9763,60 @@ components:
           description: Number of strings not added to the translationJob from this file
           type: integer
       type: object
+    AddFileToBatchAsyncResponse:
+      type: object
+      required:
+      - response
+      properties:
+        response:
+          allOf:
+            - $ref: '#/components/schemas/AsyncAcceptedResponse'
+            - properties:
+                data:
+                  properties:
+                    fileUri:
+                      description: Uri of the file linked to the Batch
+                      type: string
+                    batchUid:
+                      description: The UID of the batch
+                      type: string
+                    batchItemId:
+                      description: The id of the batch item
+                      type: integer
+    BatchItemStatusResponse:
```

Comment:
I would call it `JobBatchItemStatusResponse` to make it clearer what kind of batch item it's about